index.js

```diff
@@ -435,13 +435,16 @@ export default class OneSignal {
         return RNOneSignal.userProvidedPrivacyConsent();
     }
 
-    static setExternalUserId(externalId, callback) {
+    static setExternalUserId(externalId, varArg1, varArg2) {
         if (!checkIfInitialized()) return;
 
-        if (callback === undefined)
-            callback = function(){};
+        if (typeof varArg1 === "function") {
+            RNOneSignal.setExternalUserId(externalId, null, varArg1);
+            return;
+        }
+
+        RNOneSignal.setExternalUserId(externalId, varArg1, varArg2 || function(){});
```

Comment:
I imagine this exists on ES5 but not 100 sure

Reply:
yes, this works in ES5